File: gluecodium/src/test/resources/smoke/dates/output/android/com/example/smoke/DateDefaults.java
/*
 *
 */
package com.example.smoke;
import android.support.annotation.NonNull;
import java.util.Date;
public final class DateDefaults {
    @NonNull
    public Date dateTime;
    @NonNull
    public Date dateTimeUtc;
    @NonNull
    public Date beforeEpoch;
    @NonNull
    public Date exactlyEpoch;
    public DateDefaults() {
        this.dateTime = new Date(1643966117000L);
        this.dateTimeUtc = new Date(1643966117000L);
        this.beforeEpoch = new Date(-1511793883000L);
        this.exactlyEpoch = new Date(0L);
    }
    public DateDefaults(@NonNull final Date dateTime, @NonNull final Date dateTimeUtc, @NonNull final Date beforeEpoch, @NonNull final Date exactlyEpoch) {
        this.dateTime = dateTime;
        this.dateTimeUtc = dateTimeUtc;
        this.beforeEpoch = beforeEpoch;
        this.exactlyEpoch = exactlyEpoch;
    }
}

File: gluecodium/src/test/resources/smoke/dates/output/android-kotlin/com/example/smoke/DateDefaults.kt
/*

 *
 */

@file:JvmName("DateDefaults")

package com.example.smoke

import java.util.Date

class DateDefaults {
    @JvmField var dateTime: Date
    @JvmField var dateTimeUtc: Date
    @JvmField var beforeEpoch: Date
    @JvmField var exactlyEpoch: Date



    constructor() {
        this.dateTime = Date(1643966117000L)
        this.dateTimeUtc = Date(1643966117000L)
        this.beforeEpoch = Date(-1511793883000L)
        this.exactlyEpoch = Date(0L)
    }





}

